msal/src/main/java/com/microsoft/identity/client/UiBehavior.java

```diff
@@ -59,4 +61,17 @@ public String toString() {
                 throw new IllegalArgumentException();
         }
     }
+
+    public static OpenIdConnectPromptParameter mapUIBehaviorToOpenIdConnect(UiBehavior uiBehavior){
```

Comment:
This utility method doesn't feel at-home to me here. This behavior feels very adapter-like (but this _also_ doesn't seem worth making a whole new object for...)

How about an instance method on this enum called `toOpenIdConnectPromptParameter()`?

Reply:
So you could go....

`UiBehavior.SELECT_ACCOUNT.toOpenIdConnectPromptParameter()`

Reply:
I like the idea. Thanks for the suggestion Brian, adopted here